packages/cactus-plugin-bungee-hermes/src/main/typescript/view-creation/privacy-policies.ts

```diff
@@ -8,6 +8,35 @@ export interface IPrivacyPolicyValue {
   policy: PrivacyPolicyOpts;
   policyHash: string;
 }
+
+// Type guard for PrivacyPolicyOpts
+export function isPrivacyPolicyOpts(
```

Comment:
Shouldn't this changes be in a different PR as they are done in the bungee package and the name of this pr is referent to the satp one?

Reply:
I just had a talk with @petermetz (please correct me if I'm wrong), and we have these options:
- Merge everything together, since the future merge of `satp-dev` into main will be divided into multiple smaller PRs
- Merge these changes separately into `satp-dev`
- Merge these changes separately into `main` (they would be applied to `satp-dev` only after rebasing)

What do you think? (tagging @RafaelAPB so he sees this as well)

Reply:
@brunoffmateus @LordKubaya For reference, since this PR is against satp-dev, not main, I'm not too worried about keeping with the convention that @LordKubaya mentions in https://github.com/hyperledger-cacti/cacti/pull/3509#discussion_r1824588336 (if this was a PR against main then of course I'd ask for the same thing).

Reply:
Okey @petermetz thank you.